docs/content/policy-language.md

```diff
@@ -1672,39 +1631,32 @@ http_servers contains server if {
 
 ## Future Keywords
 
-To ensure backwards-compatibility, new keywords (like `every`) are introduced slowly.
-In the first stage, users can opt-in to using the new keywords via a special import:
+To ensure backwards-compatibility, new keywords (like `every`) were introduced slowly.
+In the first stage, users could opt-in to using the new keywords via a special import:
 
 * `import future.keywords` introduces _all_ future keywords, and
 * `import future.keywords.x` _only_ introduces the `x` keyword -- see below for all known future keywords.
-* `import rego.v1` introduces all future keywords, and enforces the use of `if` and `contains` in rule heads where applicable.
-
-{{< danger >}}
-Using `import future.keywords` to import all future keywords means an **opt-out of a
-safety measure**:
-
-With a new version of OPA, the set of "all" future keywords can grow, and policies that
-worked with the previous version of OPA stop working.
-
-This **cannot happen** when you selectively import the future keywords as you need them.
-{{< /danger>}}
```

Comment:
this also didn't seem to be so relevant now since there will be no new keywords

Reply:
This might still become relevant again in the future if we introduce new keywords that'll become standard in v2. But then we can add something like this back. This section is probably more confusing than helpful in the meanwhile 👍 .